### DISCUSSION TITLE
OIDC and redirect_uri

### DISCUSSION THREAD

**Discussion by ytqsl** [datalust/seq-tickets#1880] [Help] [answered]
Hi,

I'm evaluating Seq, hosted in a private OpenShift environment:
- Deployed using Helm (with some security context tweaks
- Create secured route with edge termination and https redirect
- Configured OIDC to a KeyCloak OIDC server

When Seq redirects to KeyCloak, it sets the redirect_uri to the correct callback url but for some reason it sets the protocol to http instead of https, even though in the configuration screen it shows https. I suspect Seq doesn't respect reverse proxy forward header.

Any thoughts how to get this fixed?

**Comment by KodrAus** [comment, marked as answer]
Hi @ytqsl :wave:

This is something you should be able to use [Seq's canonical URI](https://github.com/datalust/helm.datalust.co/blob/main/charts/seq/values.yaml#L23) to configure in your Helm deployment. If set, it's an address Seq will use as a base when generating inbound links. It's intended for these kinds of scenarios where Seq is somewhere behind a proxy.

**Reply by ytqsl** [reply]
That worked, thanks!
